pyvelox/test/test_vector.py

```diff
@@ -28,6 +49,18 @@ def test_from_list(self):
         with self.assertRaises(ValueError):
             pv.from_list([])
 
+    def test_constant_encoding(self):
+        vec = pv.constant_vector(1000, 10)
```

Comment:
need also tests for string , plus other types

Reply:
Added for string and null and float